### PR TITLE
Removed Total and added event's paymentCurrency in Admin Sales

### LINKED ISSUE #2316: Discrepancies in Admin Sales
**Describe the bug**

Currently there are a lot of bugs - 
 - `By Event`, `Discounted Events` and `Fees` can have `paymentCurrency` of the events. Currently its hard-coded to `US $`.
 - In above mentioned tabs, `Total` doesn't make any sense as different events can have different currencies.

**To Reproduce**
1. Go to 'Admin'
2. Click on 'Sales'
3. See error

**Expected behavior**

Sales of a particular event must have its own predefined currency and `Total` row should be removed till other alternative is finalised.

**Screenshots**

_By Events_
Screenshot_2019-03-18 Overview Sales Administration Open Event

_Fees_
Screenshot_2019-03-18 Fees Sales Administration Open Event(1)

**Desktop (please complete the following information):**
 - Ubuntu 18.04, Mozilla Firefox

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Screenshots
_By Events_
Screenshot_2019-03-18 Overview Sales Administration Open Event(1)

_Fees_
Screenshot_2019-03-18 Fees Sales Administration Open Event

#### Changes proposed in this pull request:

- Removes `Total` from Admin Sales 
- Removes hard-coded currency

Fixes #2316

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 please review. If you think fix is ok, I'll open a PR for server too.

File: app/models/admin-sales-discounted.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import attr from 'ember-data/attr';
...
 
 export default ModelBase.extend({
-  code      : attr('string'),
-  email     : attr('string'),
-  eventName : attr('string'),
-  sales     : attr()
+  code            : attr('string'),
+  email           : attr('string'),
+  eventName       : attr('string'),
+  paymentCurrency : attr('string'),
```
Comment (CosmicCoder96) [inline app/models/admin-sales-discounted.js]: is this returned by the server? Have you verified?

Comment (shreyanshdwivedi) [reply app/models/admin-sales-discounted.js]: @CosmicCoder no. I've made changes on my local server. I'll push it once you confirm this is relevant issue.

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 corresponding PR on server side - https://github.com/fossasia/open-event-server/pull/5698

File: app/models/admin-sales-discounted.js (hunk 1 above)
Comment (shreyanshdwivedi) [reply app/models/admin-sales-discounted.js]: Opened a PR on server

**Pull request merged**